.builds/freebsd.yml

```diff
@@ -7,21 +8,30 @@ packages:
   - x11-fonts/fontconfig
   - x11-fonts/dejavu
   - x11/libxcb
+
 sources:
   - https://github.com/alacritty/alacritty
+
+environment:
+  PATH: /home/build/.cargo/bin:/bin:/usr/bin:/usr/local/bin
```

Comment:
As long as it works, I don't think it matters?

Reply:
Different versions of applications aren't generally shipped multiple times in these locations on FreeBSD, right?

Reply:
>Different versions of applications aren't generally shipped multiple times in these locations on FreeBSD, right?

No, `/usr/bin` have some as well as `/bin` has some common common utils, and they also link to system libraries  from `/usr/lib`, everything from `/usr/local/bin` are linked with libraries under `/usr/local/lib`. So I'd generally make local path before everything else, just because everything is goes there, and you want to always use a tool you're installing, not the one system provides (They even has duplicated libs, since you have ncurses in local and ncurses in /usr/lib, and they're completely different).

Reply:
> and you want to always use a tool you're installing, not the one system provides

We're not talking about a user system though, but CI. What practical difference does this make?

Reply:
Also fyi, this is the default `PATH`: `/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin:/home/build/bin`.

So this matches what I have.